### DISCUSSION TITLE
Ceph Objectstore Multisite with cloud sync module

### DISCUSSION THREAD

**Discussion by WoodyWoodsta** [rook/rook#13717] [Q&A] [answered]
Hey all!

We have an ObjectStore which we would like to replicate off-site (AWS) for the purpose of backups. There are numerous ways this can be done, but we've found that the most Ceph-ecosystem-appropriate way is to setup another zone within the zone-group in the ObjectStore and configure the second zone to be of tier-type "cloud", as described in [the Ceph documentation](https://docs.ceph.com/en/latest/radosgw/cloud-sync-module/).

However, I'm not sure how this is practically achieved with Rook. From my (potentially limited) understanding of Ceph and RGW, in order to have a functioning replication zone, even if it uses an off-site backing storage provider, you need another gateway. Using rook, this appears only achievable by creating a `CephObjectZone` CRD, which will configure the cluster with another zone, and another gateway. The only problem with this, is that it appears to configure the zone with tier type `rgw`, or equivalent, such that it becomes another in-cluster zone (with metadata 
 and data pools).

How should I go about configuring the cloud-sync zone such that it does not start to replicate data within the rook cluster itself? Have I misunderstood the configurability or lifecycle of a `CephObjectZone`?

**Comment by travisn** [comment]
Besides the [Object store multisite documentation](https://rook.io/docs/rook/latest/Storage-Configuration/Object-Storage-RGW/ceph-object-multisite/), @thotz any input?

**Comment by thotz** [comment, marked as answer]
cloud sync module is not added yet, please file RFE in Rook, Not sure whether we can add existing zone,zonegroup crds or introduce a new one

**Reply by WoodyWoodsta** [reply]
Will do!
